src/database/factories/SourceFactory.php

```diff
@@ -20,7 +20,7 @@ public function definition()
         return [
             'name' => $this->faker->name,
             'source_type_id' => ListSourceType::inRandomOrder()->first()->id,
-            'trust' => $this->faker->randomFloat(2, 0, 10),
+            'trust' => 1,
```

Comment:
Why is this better than using a random number ?

Reply:
Because the trust feature isn't available yet. The controller/request aren't coded to support trust. Then the database is fill with 1 by default.
Having a random number here will cause error in test since the controller can't handle numbers.